iree/vm/ops.h

```diff
@@ -318,6 +318,16 @@ static inline int32_t vm_cast_f32si32(float operand) {
 static inline int32_t vm_cast_f32ui32(float operand) {
   return (uint32_t)lroundf(operand);
 }
+static inline float vm_bitcast_i32f32(int32_t operand) {
+  float result;
+  memcpy(&result, &operand, 4);
```

Comment:
`sizeof(result)` here and below is a good convention (preferring the dest so that you can't write more memory than the destination)